File: XOutput.Api/Websocket/Xbox/XboxFeedbackResponse.cs
namespace XOutput.Websocket.Xbox
{
    public class XboxFeedbackResponse : MessageBase
    {
        public const string MessageType = "XboxFeedback";

        public XboxFeedbackResponse()
        {
            Type = MessageType;
        }

        public double SmallForceFeedback { get; set; }
        public double BigForceFeedback { get; set; }
        public int LedNumber { get; set; }
    }
}

File: @xoutput/api/src/websocket/xbox/XboxFeedbackResponse.ts
import { MessageBase } from '../MessageBase';

export const XboxFeedbackResponseType = 'XboxFeedback';

export interface XboxFeedbackResponse extends MessageBase<typeof XboxFeedbackResponseType> {
  smallForceFeedback: number;
  bigForceFeedback: number;
  ledNumber: number;
}
